invenio_app_ils/circulation/templates/invenio_app_ils_circulation/notifications/bulk_extend.html

```diff
@@ -26,18 +26,18 @@
 Dear {{ patron.name }}, <br/><br/>
 
 {% if extended_loans %}
-your loans have been extended as follows:
+your loans have been extended as follows:<br/>
 {% for obj in extended_loans %}
 <a href="{{ spa_routes.HOST }}{{ spa_routes.PATHS['literature']|format(pid=obj.document.pid) }}">"{{ obj.document.full_title }}"</a> until <b>{{ obj.loan.end_date }}</b><br/>
-{% endfor %}<br/><br/>
+{% endfor %}
 {% endif %}
 
 {% if not_extended_loans %}
-your loans for: <br/><br/>
+{% if extended_loans %}<br/> {% endif %}
```

Comment:
you can move this `<br>` just above, between endfor and endif and remove it from here:
```
{% endfor %}
<br />
{% endif %}
```